src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+public class DataStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
+
+    private Logger logger = LoggerFactory.getLogger(DataStoreTestAutomationAdapter.class);
+
+    private static final String[] TEST_DEFINITIONS_NEEDED_FOR_TA = {
+            "src/aat/resources/CCD_CNP_27.xlsx",
```

Comment:
wouldn't it be better to have this as config rather than hardcoded?

Reply:
or can't you remove this at all by changing the framework to pick and import any .xlsx fine under that dir?